src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The excavator has just started working");

```

Comment:
The message in the 'doWork' method should indicate the type of machine more informatively. Consider specifying that it is an 'Excavator' that started working, to make the message clear and informative.